content/contribute/writing-style-guide.md

```diff
@@ -4,33 +4,118 @@ weight: 10
+  * Describe link text improves accessibility for screen readers.
 * Order brand names alphabetically. For example, AWS, Azure, GCP.
-* Don't use smart quotes (“like this”). Only use standard straight quotes ("like this").
+  * Ordered names removes the appearance of preference.
+* Don't use "easy," "simple," or "obvious".
+  * It's condescending to the reader.
+* No Oxford commas.
+  * A subjective style choice. No commas before "and" or "or."
+* U.S. English spelling and grammar.
 <!-- vale on -->
 
-Crossplane documentation is adopting
-[Vale](https://github.com/errata-ai/vale) and relies on the [Upbound Vale
-definitions](https://github.com/upbound/vale) for style guidelines.
+Crossplane relies on [Vale](https://github.com/errata-ai/vale) to enforce the
+complete style guide. 
+
+Read more about [using Vale with the Crossplane
+documentation]({{<ref "Vale" >}}).
+
+## Italics
+
+Use _italics_ to introduce or draw attention term. 
+
+Use italics on the same term sparingly.
+
+
+## Inline code styles
+
+Use inline code styles, single ticks (`` ` ``) for files directories or paths. 
+
+Use the `{{</* hover */>}}` shortcode to relate command explanations to larger examples. 
+
+## Placeholders
+
+Use angle brackets (`< >`) for placeholders. Use a short, descriptive name
+inside the brackets. Use underscores between words to simplify selections.
+
+For example, to creating AWS credentials prompts for the key and secret key. 
+
+```ini {copy-lines="all"}
+[default]
+aws_access_key_id = <aws_access_key>
+aws_secret_access_key = <aws_secret_key>
+```
+## Styling Kubernetes objects
+
+### Kinds
+
+Kinds should be upper camel case. Capitalize each word, without separators. 
+
+<!-- vale Google.FirstPerson = NO -->
+<!-- ignore "My" -->
+For example the 
+{{<hover label="kind" line="4" >}}kind: MyComputeResource{{< /hover >}} 
+capitalizes "My," "Compute" and "Resource" with no spaces or dashes.
+<!-- vale Google.FirstPerson = YES -->
+
+```yaml {label="kind"}
+spec:
+  group: test.example.org
+  names:
+    kind: MyComputeResource
+```
+
+### Names
+
+Object names should use snake case. All words are lowercase with dashes (`-`)
+between them.
+
+<!-- vale Google.FirstPerson = NO -->
+<!-- ignore "My" -->
+For example the 
+{{<hover label="kind" line="4" >}}name: my-resource{{< /hover >}} 
+uses all lowercase. A dash separates "my" and "resource."
+<!-- vale Google.FirstPerson = YES -->
+
+```yaml {label="name"}
+apiVersion: test.example.org/v1alpha1
+kind: MyComputeResource
+metadata:
+  name: my-resource
+```
 
-Beyond Vale, Crossplane recommends [Grammarly](https://www.grammarly.com/) and [Hemingway
-Editor](https://hemingwayapp.com/) to improve writing quality.
+### Inline
 
-The Crossplane maintainers recommend the [Google developer documentation style guide](https://developers.google.com/style) 
-to help answer questions on proper styling. 
+Kubernetes objects mentioned inline don't require styling unless drawing special
```

Comment:
```suggestion
<!-- vale off -->
<!-- turn off vale checking for this line as special doesn't refer to a person. [alex.Ablist] -->
Kubernetes objects mentioned inline don't require styling unless drawing special
<!-- vale on -->


```

Reply:
I would also consider whether we should remove the rule for `special`, since it is highly unlikely we would reference people with disabilities in our documentation.

Reply:
+1. Committing a fix to remove that check. This is exactly the kind of "figuring out the guide/checks" we need to work though.

Reply:
Fixed in style update. aa43e183d282b2388a794b9775655c049591f2cb

Confirmed via updated CI run.